ocaml/seahash/hash.ml

```diff
@@ -0,0 +1,16 @@
+open Ctypes
+open Foreign
+
+let bytes = foreign "bytes" (ocaml_bytes @-> uint64_t @-> returning uint64_t)
+let file = foreign "file" (string @-> returning uint64_t)
```

Comment:
Like I said below, I think this should take a file descriptor or `FILE` object, not a file name. This makes the function more flexible (can read from a pipe, for example) and dealing with opening and closing is left to the caller.